### PR TITLE
added a washing dishes chore

### PR DESCRIPTION
**Pull request by mitia-anah**
Hello team,

please have a look at my PR.I added one file,please check it and  change it if need be.

Thank you .

### CODE REVIEW

File: washing-dishes .md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Whashing dishes
```
Comment (moro-patana) [inline washing-dishes .md]: oops! your speliing is wrong

**Review by moro-patana — CHANGES_REQUESTED** [review]

File: washing-dishes .md (hunk 1 above)
Comment (mitia-anah) [reply washing-dishes .md]: can find my mistake

Comment (mitia-anah) [reply washing-dishes .md]: what is my mistake pleas?

Comment (mitia-anah) [inline washing-dishes .md]: Oops ,sorry about the mistake on my file name,it should not have space.

Comment (mitia-anah) [inline washing-dishes .md]: Thanks for checking my PR.

Comment (mitia-anah) [reply washing-dishes .md]: Thank you dear for checking my PR.

**Review by mitia-anah — COMMENTED** [review]
Please tell me what the mistake was.

**Review by Ny-Rianala — APPROVED** [review]

**Comment by Ny-Rianala** [comment]
> Hello team,
> 
> please have a look at my PR.I added one file,please check it and change it if need be.
> 
> Thank you .

**Comment by Ny-Rianala** [comment]
Looks good?

**Pull request merged**